src/cmd-buildextend-installer

```diff
@@ -206,17 +210,47 @@ def generate_iso():
         genisoargs += ['-r', '-l', '-sysid', 'PPC',
                        '-chrp-boot', '-graft-points']
     elif basearch == "s390x":
+        INITRD_ADDRESS = '0x02000000'
+        lorax_templates = '/usr/share/lorax/templates.d/99-generic/config_files/s390'
+        shutil.copy(os.path.join(lorax_templates, 'redhat.exec'), tmpisoimages)
+        with open(os.path.join(lorax_templates, 'generic.ins'), 'r') as fp1:
+            with open(os.path.join(tmpisoroot, 'generic.ins'), 'w') as fp2:
+                [fp2.write(line.replace('@INITRD_LOAD_ADDRESS@', str(INITRD_ADDRESS))) for line in fp1]
```

Comment:
`INITRD_ADDRESS` is already a string, no?

Reply:
Right, it's already a string. A little bit redundant though.

Reply:
Changed in latest commit.